compclub/compclub/settings.py

```diff
@@ -124,5 +124,4 @@
 
 STATIC_URL = '/static/'
 
-LOGIN_REDIRECT_URL = 'website:index'
```

Comment:
yes, cuz `LOGIN_REDIRECT_URL`'s default is set to `accounts/profile/`(as seen from [here](https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-LOGIN_REDIRECT_URL)), so i bind a view into the default URL instead of using the previous `login/`

https://github.com/WeilonYing/compclub-web/blob/07a101b5b0f2c5dacec2eedbc3bc4fdf78eccf73/compclub/website/urls.py#L10

https://github.com/WeilonYing/compclub-web/blob/07a101b5b0f2c5dacec2eedbc3bc4fdf78eccf73/compclub/website/views.py#L35-L38

should i change it back to `login/`?

Reply:
Ah fair enough. No need to change that then. Go ahead and merge.